src/Microsoft.ML.GenAI.Core/Extension/ModuleExtension.cs

```diff
@@ -90,13 +90,18 @@ public static void ToInt8QuantizeModule<T>(
     /// </summary>
     /// <typeparam name="T"></typeparam>
     /// <param name="model"></param>
-    public static void ToInt4QuantizeModule<T>(
-        this T model)
+    /// <param name="quantizedDType">Quantized data type, can be "fp4" or "nf4".</param>
```

Comment:
Maybe either here or in the summary add a note as to the difference between "fp4" and "nf4".